src/merchantPayment/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum PaymentDeepLinkHandler {
```

Comment:
this type is a little odd, should we just have a boolean `merchantPaymentEnabled` remote config variable instead? we can add more variants if we use an enum but seems unlikely that we want to build on this?